### ISSUE TITLE
[Update from 1.5 to last version 7.0.x] Space banner are no more displayed after update

### ISSUE THREAD

**Issue by rdenarie** [Meeds-io/meeds#2689]
**Steps to reproduce**

1. Start a version meeds 1.5.x
2. Create spaces, and add a banner for these spaces
3. Migrate to 7.0.x
4. Open spaces

**Current Behaviour**
Space banner are not displayed.

When going to space administration, and apply a template to the space, the banner  is correctly displayed, but this action is manual.

When upgrading a customer, we have hundreds of spaces to migrate, and it is not possible to do it manually

**Comment by boubaker** [comment]
> When upgrading a customer, we have hundreds of spaces to migrate, and it is not possible to do it manually

You can Apply Template in Bulk in Fact, @srenault-meeds WDYT ?

**Comment by srenault-meeds** [comment]
Indeed, this is why I didn't suggest to create an UP for this banner case.

**Comment by srenault-meeds** [comment]
Ok. after discussing it with @margondicco, it is needed to study an UP to ease the process of migrating from 1.5x to 7.0x.

Expected behaviour (Suggestion)
Space menu portlet (old portlet) must be replaced by space banner (new portlet)

Let me know if you have any other idea

**Comment by boubaker** [comment]
> Space menu portlet (old portlet) must be replaced by space banner (new portlet)

**Only** in the context of the **home page**, because the old version of menu was added in **all spaces pages** but hidden programatically (legacy from 5.x versions).
